reco_utils/common/python_utils.py

```diff
@@ -2,29 +2,50 @@
 from scipy.sparse import coo_matrix
 
 
+def exponential_decay(value, max_val, half_life):
+    """Compute decay factor for a given value based on an exponential decay
+    Args:
+        value (numeric): value to calculate decay factor
```

Comment:
We are not using the notation numeric in the rest of the code, we use np.array, float, int...